docs/_documentations/remote-deploying-codewind.md

```diff
@@ -122,7 +161,7 @@ devex001   jane       codewind    119m   devex001   Complete       https://codew
 
 ## 5. Removing a Codewind instance
 
-To remove a Codewind instance, see [Removing a Codewind instance](https://github.com/eclipse/codewind-operator/blob/master/README.md#removing-a-codewind-instance) in the codewind-operator readme.
+To remove a Codewind instance, see [Removing a Codewind instance](https://github.com/eclipse/codewind-operator/blob/master/README.md#removing-a-codewind-instance) in the codewind-operator readme file.
```

Comment:
The title and hyperlink, "Removing a Codewind instance," suffices here. I think we can delete the "in the codewind-operator readme file" part.

Reply:
Corrected.